tools/topology/sof/pipe-volume-demux-playback.m4

```diff
@@ -23,13 +23,12 @@ include(`mixercontrol.m4')
 include(`bytecontrol.m4')
 
 # demux Bytes control with max value of 255
-C_CONTROLBYTES(DEMUX, PIPELINE_ID,
+C_CONTROLBYTES(concat(`DEMUX', PIPELINE_ID), PIPELINE_ID,
```

Comment:
does this impact userspace, e.g. with a change of control names?

Reply:
Good question. Let me check

Reply:
The name will have some little difference, but no influence.